MapleServer2/PacketHandlers/Game/ClubHandler.cs

```diff
@@ -84,7 +96,7 @@ private void HandleJoin(GameSession session, PacketReader packet)
             {
                 foreach (Player member in party.Members)
                 {
-                    party.BroadcastPacketParty(ClubPacket.UpdatePlayerClubList(member, club.Id));
+                    party.BroadcastPacketParty(ClubPacket.UpdatePlayerClubList(session.Player, club));
```

Comment:
This seems wrong, this is making the packet send multiple times but only with the same person